compiler-java/src/com/redhat/ceylon/ceylondoc/CeylonDocTool.java

```diff
@@ -892,8 +892,7 @@ private void copySourceFiles() throws FileNotFoundException, IOException {
                 markup.around("script src='" + getResourceUrl(pkg, "ceylondoc.js") + "' type='text/javascript'"); 
                 markup.close("head");
                 markup.open("body", "pre data-language='ceylon' style='font-family: Inconsolata, Monaco, Courier, monospace'");
-                // XXX source char encoding
-                BufferedReader input = new BufferedReader(new InputStreamReader(pu.getUnitFile().getInputStream()));
+                BufferedReader input = new BufferedReader(new InputStreamReader(pu.getUnitFile().getInputStream(), getEncoding() != null ? getEncoding() : CeylonConfig.get(DefaultToolOptions.DEFAULTS_ENCODING)));
```

Comment:
I think the decision was made a long time ago that if we can't force Microsoft and Apple to default to UTF8 we shouldn't either (or people will get unexpected results when editing and compiling on the command line). Some of us said "f* them" but we were overruled ;)